.swiftlint.yml

```diff
@@ -0,0 +1,4 @@
+disabled_rules: # rule identifiers to exclude from running
+  - trailing_whitespace
```

Comment:
Whitespace between functions is fine, I don't see why the trailing whitespace is needed though.